app/dependencies.py

```diff
@@ -22,13 +23,30 @@ async def get_users(db: AsyncSession = Depends(get_db)):
 from app.database import SessionLocal
 from app.models import models_core
 from app.schemas import schemas_auth
+from app.utils.redis import connect
 from app.utils.tools import is_user_member_of_an_allowed_group
 from app.utils.types.groups_type import GroupType
 from app.utils.types.scopes_type import ScopeType
 
 # We could maybe use hyperion.security
 hyperion_access_logger = logging.getLogger("hyperion.access")
 
+redis_client = None  # Create a global variable for the redis client, so that it can be instancied in the startup and shutdown events
```

Comment:
Could you type `redis_client`?

```suggestion
redis_client: redis.Redis | bool | None = None  # Create a global variable for the redis client, so that it can be instancied in the startup and shutdown events
```